src/test/java/com/backend/blooming/stamp/presentation/StampControllerTest.java

```diff
@@ -241,4 +252,358 @@ class StampControllerTest extends StampControllerTestFixture {
+                .header("X-API-VERSION", 1)
+                .header(HttpHeaders.AUTHORIZATION, 액세스_토큰)
+                .with(request -> {
+                    request.setMethod("PATCH");
+                    return request;
+                })
+        ).andExpectAll(
+                status().isOk(),
+                jsonPath("$.id", is(수정한_스탬프_응답_dto.id()), Long.class),
+                jsonPath("$.userName", is(수정한_스탬프_응답_dto.userName())),
+                jsonPath("$.userColor", is(수정한_스탬프_응답_dto.userColor())),
+                jsonPath("$.day", is(수정한_스탬프_응답_dto.day()), long.class),
+                jsonPath("$.message", is(수정한_스탬프_응답_dto.message())),
+                jsonPath("$.stampImageUrl", is(수정한_스탬프_응답_dto.stampImageUrl()))
+        ).andDo(print()).andDo(restDocs.document(
+                pathParameters(
+                        parameterWithName("goalId").description("스탬프를 수정할 골 아이디"),
+                        parameterWithName("stampId").description("수정할 스탬프 아이디")
+                ),
+                requestHeaders(
+                        headerWithName("X-API-VERSION").description("요청 버전"),
+                        headerWithName(HttpHeaders.AUTHORIZATION).description("액세스 토큰")
+                ),
+                requestParts(
+                        partWithName("request").description("수정할 스탬프 정보"),
+                        partWithName("stampImage").description("수정할 스탬프 이미지")
+                ),
+                responseFields(
+                        fieldWithPath("id").type(JsonFieldType.NUMBER).description("수정한 스탬프 골 아이디"),
+                        fieldWithPath("userName").type(JsonFieldType.STRING).description("스탬프를 수정한 사용자 이름"),
+                        fieldWithPath("userColor").type(JsonFieldType.STRING).description("스탬프를 수정한 사용자 테마 색상코드"),
+                        fieldWithPath("day").type(JsonFieldType.NUMBER).description("수정한 스탬프 날짜"),
+                        fieldWithPath("message").type(JsonFieldType.STRING).description("수정한 스탬프 메시지"),
+                        fieldWithPath("stampImageUrl").type(JsonFieldType.STRING).description("수정한 스탬프 이미지 url")
+                )
+        ));
+    }
+
+    @Test
+    void 스탬프_메시지만_수정한다() throws Exception {
+        // given
+        given(tokenProvider.parseToken(액세스_토큰_타입, 액세스_토큰)).willReturn(사용자_토큰_정보);
+        given(userRepository.existsByIdAndDeletedIsFalse(사용자_토큰_정보.userId())).willReturn(true);
+        given(stampService.update(유효한_사용자_아이디, 유효한_골_아이디, 메시지만_수정하는_스탬프_수정_dto)).willReturn(메시지만_수정한_스탬프_dto);
+
+        // when & then
+        mockMvc.perform(multipart("/goals/{goalId}/stamp/{stampId}", 유효한_골_아이디, 유효한_스탬프_아이디)
+                .file(스탬프_이미지가_null인_파일)
+                .file(메시지만_수정하는_스탬프_수정_요청)
+                .contentType(MediaType.MULTIPART_FORM_DATA)
+                .header("X-API-VERSION", 1)
+                .header(HttpHeaders.AUTHORIZATION, 액세스_토큰)
+                .with(request -> {
+                    request.setMethod("PATCH");
```

Comment:
별건 아니지만, 실수를 줄이기 위해 아래처럼 수정하면 어떨까요?
```suggestion
                    request.setMethod(HttpMethod.PATCH.name());
```